data_steward/tools/create_tier.py

```diff
@@ -58,6 +59,92 @@ def validate_release_tag_param(arg_value):
     return arg_value
 
 
+def create_datasets(client, name, input_dataset, tier, release_tag, deid_stage):
+    """
+    Creates backup, staging, sandbox, and final datasets with the proper descriptions
+    and tag/labels applied
+
+    :param client: an instantiated bigquery client object
+    :param name: the base name of the datasets to be created
+    :param input_dataset: name of the input dataset
+    :param tier: tier parameter passed through from either a list or command line argument
+    :param release_tag: release tag parameter passed through either the command line arguments
+    :param deid_stage: deid_stage parameter passed through from either a list or command line argument
+    :return: tuple of created dataset names
+    """
+
+    if not client:
+        raise RuntimeError("Please specify BigQuery client object")
+    if not name:
+        raise RuntimeError(
+            "Please specify the base name of the datasets to be created")
+    if not input_dataset:
+        raise RuntimeError("Please specify the name of the input dataset")
+    if not tier:
+        raise RuntimeError(
+            "Please specify the tier intended for the output datasets")
+    if not release_tag:
+        raise RuntimeError(
+            "Please specify the release tag for the dataset in the format of YYYY#q#r"
+        )
+    if not deid_stage:
+        raise RuntimeError(
+            "Please specify the deid stage (deid, base, or clean)")
+
+    # Construct names of datasets need as part of the deid process
+    final_dataset_id = name
+    backup_dataset_id = f'backup_{name}'
+    staging_dataset_id = f'staging_{name}'
+    sandbox_dataset_id = f'sandbox_{name}'
```

Comment:
Ah, I see. Yes, I did do this because the names in the ticket, I took them at face value 🙈however, in the [Controlled Tier Curation Architecture](https://docs.google.com/document/d/1GKESjTzVNjC2WPuMDMQl96l5EFS1Y-ICxiP5d6YkPrI/edit#heading=h.adbimkeolw6g) documentation there is a function that will be created called `get_dataset_name` which will take the tier, release_tag, and suffix and return `{tier_prefix}{release_tag}_{suffix}`. This will end up being the passed in `name` parameter. So to fix this, I should change the string to `f'{name}_backup`, `f'{name}_staging`, `f'{name}_sandbox`?

Reply:
Yes! These should invoke that function for the names. The constants for `backup` etc. could facilitate this.